llvm/test/CodeGen/RISCV/rvv/vl-opt-instrs.ll

```diff
@@ -2929,21 +2929,17 @@ define <vscale x 4 x i32> @vid.v(<vscale x 4 x i32> %c, iXLen %vl) {
 define <vscale x 4 x float> @vfadd_vv(<vscale x 4 x float> %a, <vscale x 4 x float> %b, iXLen %vl) {
 ; NOVLOPT-LABEL: vfadd_vv:
 ; NOVLOPT:       # %bb.0:
-; NOVLOPT-NEXT:    fsrmi a1, 0
-; NOVLOPT-NEXT:    vsetvli a2, zero, e32, m2, ta, ma
+; NOVLOPT-NEXT:    vsetvli a1, zero, e32, m2, ta, ma
```

Comment:
The removal of the fsrmi here belongs in the previous commit.

Reply:
Updated.